CIScripts/Test/Tests/Protocol/SingleHostWithUtils.Tests.ps1

```diff
@@ -137,7 +137,14 @@ Describe "Single compute node protocol tests with utils" {
                 Remove-Variable "ContrailNetwork"
             }
         } finally {
-            Merge-Logs -LogSources (New-LogSource -Path (Get-ComputeLogsPath) -Sessions $Session)
+            [LogSource[]] $LogSources = New-FileLogSource -Path (Get-ComputeLogsPath) -Sessions $Session
+            foreach ($ContainerVar in "Container1ID", "Container2ID") {
```

Comment:
This procedure seems to be more complicated than necessary. Can we handle container names/ids in the same manner as in TunnellingWithAgent.Tests.ps1? In those tests container names are fixed.